src/Symfony/Component/Validator/Constraints/NullValidator.php

```diff
@@ -18,6 +18,8 @@
  * @author Bernhard Schussek <bschussek@gmail.com>
  *
  * @api
+ *
+ * @deprecated
```

Comment:
It's actually not.. Should we still add NullConstraintValidator for consistency though?